README.md

```diff
@@ -20,3 +20,7 @@ The user of the app will be able to:
 
 ## Installation
 coming soon
+
+
+
+[![Coverage Status](https://coveralls.io/repos/github/ljte/lab-project/badge.svg?branch=working_branch)](https://coveralls.io/github/ljte/lab-project?branch=working_branch)
```

Comment:
When you create a pull request, your [Build Status] and [Coverage Status] should point to the master branch.

Reply:
done